### DISCUSSION TITLE
Make folders better visible

### DISCUSSION THREAD

**Discussion by t-arn** [gsantner/markor#1557] [General] [answered]
The folders should be more clearly distinguishable from files. Maybe with a different color?

**Comment by gsantner** [comment, marked as answer]
There is a option in the toolbar to always show folders on top. This way you get a grouping right away, all folders first, then the files.

I experiment with a few icons and colors and it always felt worse, distracting and bloat. For the real important stuff (also folders), you can favorite them, and yellow color is used for highlighting.

**Reply by aisbergde** [reply]
But if icons have no additional information about the file type, then they could / should wast less space. If they are there, they could contain information about the file type. Normally they have the same type in my notes (adoc), but sometimes I have different types (md, pdf, xml, txt) and an optical indicator would be fine. The main issue is: the file type is not shown for long filenames. Then it is a kind of trial and error to get the right file.

The following is a normal situation in my files. I edit files in AsciiDoc, convert them via docbook (xml) into markdown. And sometimes I need them also exported into pdf.

- long-filename-aaa-bbb-ccc.adoc
- long-filename-aaa-bbb-ccc.md
- long-filename-aaa-bbb-ccc.xml
- long-filename-aaa-bbb-ccc.pdf

Line break in filenames would also solve the issue.

**Reply by gsantner** [reply]
I thought about allowing multi-line expansion of filenames for a while, but didn't make a decision.

Now made the change, with the next update (v2.8.5 / v2.9.0):
The folder/filenames will then be shown fully, expanded to multiple lines.
